### PR TITLE
Add 2nd Trino teaser

### PR DESCRIPTION
**Pull request by bitsondatadev**

### CODE REVIEW

File: _posts/2022-10-18-trino-summit-2022-teaser-2.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+I can't help but bring up the analogy of how Trino federates heterogeneous data
+while this Trino Summit will federate many of us in the community form all
+corners of the earth. It really brings an appreciation to the international
+reach this technology has and makes me look forward to future in-person events!
+
+I urge you to join us either in-person or virtually if you are a fan of Trino,
+big data, open source, data engineering, Java, or all the above! This conference
+is free and takes place on November 10th. Check out the details below!
+
+<!--more-->
```
Comment (mosabua) [inline _posts/2022-10-18-trino-summit-2022-teaser-2.md:24]: probably move this up one paragraph

File: _posts/2022-10-18-trino-summit-2022-teaser-2.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+author: "Brian Olsen"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-summit-2022/summit-logo.png
+---
+
+Trino has long been the de facto standard to querying large data sets over your
+cloud or on-prem storage, also known as data lakes. This Trino Summit's theme 
+instead will showcase Trino's other claim to fame: query federation. In essence,
+Trino is a query engine providing an access point that exposes ANSI SQL across 
+[multiple data sources]({{site.url}}/docs/current/connector/).
```
Comment (mosabua) [inline _posts/2022-10-18-trino-summit-2022-teaser-2.md:13]:
```suggestion
[multiple data sources]({{site.url}}/docs/current/connector.html).
```

File: _posts/2022-10-18-trino-summit-2022-teaser-2.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+
+I can't help but bring up the analogy of how Trino federates heterogeneous data
```
Comment (mosabua) [inline _posts/2022-10-18-trino-summit-2022-teaser-2.md:15]: Thats awfully long for a filler maybe shorten and collapse with topic of next paragraph.

File: _posts/2022-10-18-trino-summit-2022-teaser-2.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+
+I urge you to join us either in-person or virtually if you are a fan of Trino,
+big data, open source, data engineering, Java, or all the above! This conference
+is free and takes place on November 10th. Check out the details below!
+
+<!--more-->
+
+## Register for the summit
+
+Trino Summit is being held at the Commonwealth Club in San Francisco, CA.
```
Comment (mosabua) [inline _posts/2022-10-18-trino-summit-2022-teaser-2.md:28]:
```suggestion
Trino Summit is held at the Commonwealth Club in San Francisco, CA.
```

File: _posts/2022-10-18-trino-summit-2022-teaser-2.md [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+Make sure you register quickly for in-person registration, as it is limited to
+250 seats! Virtual registration is also picking up quickly and is limited to
+1500 registrants. Register today before spots run out!
```
Comment (mosabua) [inline _posts/2022-10-18-trino-summit-2022-teaser-2.md:31]: Do we really want to say that virtual is limited?

File: _posts/2022-10-18-trino-summit-2022-teaser-2.md [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+    <a class="btn btn-pink" href="https://www.starburst.io/info/trinosummit/">
+        Register now
+    </a>
+</div>
+<div class="spacer-30"></div>
+
+### Trino Summit 2022 teaser
+
+Check out the teaser for this year's Trino SUmmit and get ready to Federate 'em
+all!
```
Comment (mosabua) [inline _posts/2022-10-18-trino-summit-2022-teaser-2.md:43]: maybe highlight "Federate'em all" as italics or even bold

File: _posts/2022-10-18-trino-summit-2022-teaser-2.md [hunk 7]
```diff
@@ -0,0 +1,141 @@
...
+
+<iframe src="https://www.youtube.com/embed/o2MJvRKG14M" width="800" height="500" frameborder="0" marginwidth="0" marginheight="0" scrolling="no" style="border:1px solid #CCC; border-width:1px;
+margin-bottom:5px; max-width: 100%;" allowfullscreen="">
+</iframe>
+
+
```
Comment (mosabua) [inline _posts/2022-10-18-trino-summit-2022-teaser-2.md:49]: remove empty line

File: _posts/2022-10-18-trino-summit-2022-teaser-2.md [hunk 8]
```diff
@@ -0,0 +1,141 @@
...
+## Announcing the second round of sessions and speakers
+
+As we announced in the [previous summit teaser]({% post_url
+2022-09-22-trino-summit-2022-teaser %}), we have already announced some of our
```
Comment (mosabua) [inline _posts/2022-10-18-trino-summit-2022-teaser-2.md:53]: reword ... repeated "announce"

File: _posts/2022-10-18-trino-summit-2022-teaser-2.md [hunk 9]
```diff
@@ -0,0 +1,141 @@
...
+exciting lineup of speakers! The topics range from architectures like data mesh
```
Comment (mosabua) [inline _posts/2022-10-18-trino-summit-2022-teaser-2.md:54]: dont use "data mesh" unless its really in the topic of the talk..  its just a marketing buzzword

File: _posts/2022-10-18-trino-summit-2022-teaser-2.md [hunk 10]
```diff
@@ -0,0 +1,141 @@
...
+and data lakehouse, to running Trino at scale with fault-tolerant execution, and
+of course, query federation. 
+
+We have a full roster planned but here is a glance at a few full confirmed
```
Comment (mosabua) [inline _posts/2022-10-18-trino-summit-2022-teaser-2.md:58]: where is "here" .. 

also "fully confirmed"

We have a full schedule planned for the day. Following are details on another batch of confirmed sessions.

File: _posts/2022-10-18-trino-summit-2022-teaser-2.md [hunk 11]
```diff
@@ -0,0 +1,141 @@
...
+sessions. Stay tuned for future blog posts as we announce more session as they
+are confirmed!
+
+### SK Telecom's journey to Iceberg
```
Comment (mosabua) [inline _posts/2022-10-18-trino-summit-2022-teaser-2.md:62]: Should we review/update from here on or just leave it as the original from the presenters? I think leave..  but maybe still check for grammar?

File: _posts/2022-10-18-trino-summit-2022-teaser-2.md [hunk 12]
```diff
@@ -0,0 +1,141 @@
...
+causing some jobs to fail. This can be problematic as Trino's resource sharing
+architecture can affect multiple users when a heavy query occurs.
+
+To address this situation, we optimized the data structure, tuned query, and
+used the resource group to separate them, but none of this fixed the problem. We
+learned about the advantages to using Apache Iceberg, and realized this could
+address some of these scaling issues we were facing. In this talk, we will share
+our journey.
+
+JaeChang Song, Data Engineer at SKTelecom and Trino/Iceberg Contributor
```
Comment (mosabua) [inline _posts/2022-10-18-trino-summit-2022-teaser-2.md:82]: Maybe use a bullet list for presenters and use italics ?

File: _posts/2022-10-18-trino-summit-2022-teaser-2.md [hunk 13]
```diff
@@ -0,0 +1,141 @@
...
+but none met our needs. In this session you will learn how we built a data
+architecture to collect, model, and enrich user behavior events to optimize
+Trino query performance that accelerated our ability to understand and improve
+user conversion rates.
+
+Roy Hasson, Head of Product at Upsolver
+
+## Conclusion
+
+Stay tuned for new developments in upcoming blog posts, don't forget to
```
Comment (mosabua) [inline _posts/2022-10-18-trino-summit-2022-teaser-2.md:139]: Somehow say that the summit will be great and we will forward to seeing everyone.. and then the rest

**Review by mosabua — CHANGES_REQUESTED** [review]

File: _posts/2022-10-18-trino-summit-2022-teaser-2.md (hunk 3 above)
Comment (bitsondatadev) [reply _posts/2022-10-18-trino-summit-2022-teaser-2.md:15]: Moving this under the Register section.

File: _posts/2022-10-18-trino-summit-2022-teaser-2.md (hunk 4 above)
Comment (bitsondatadev) [reply _posts/2022-10-18-trino-summit-2022-teaser-2.md:28]: I'm going to go with "will be" or "will be held". If we just say "is held" that makes it sound like it's always held at this location.

File: _posts/2022-10-18-trino-summit-2022-teaser-2.md (hunk 5 above)
Comment (bitsondatadev) [reply _posts/2022-10-18-trino-summit-2022-teaser-2.md:31]: yeah prolly not. that huge number isn't gonna scare anyone lol

File: _posts/2022-10-18-trino-summit-2022-teaser-2.md (hunk 9 above)
Comment (bitsondatadev) [reply _posts/2022-10-18-trino-summit-2022-teaser-2.md:54]: Comcast is talking about it. "Moving query fabric to data mesh."

File: _posts/2022-10-18-trino-summit-2022-teaser-2.md (hunk 11 above)
Comment (bitsondatadev) [reply _posts/2022-10-18-trino-summit-2022-teaser-2.md:62]: lol I already did some wordsmithing. I didn't want to totally make it not theirs.

File: _posts/2022-10-18-trino-summit-2022-teaser-2.md (hunk 12 above)
Comment (bitsondatadev) [reply _posts/2022-10-18-trino-summit-2022-teaser-2.md:82]: I like this, done and applied to the original teaser.

File: _posts/2022-10-19-trino-summit-2022-teaser-2.md [hunk 14]
```diff
@@ -0,0 +1,143 @@
...
+layout: post
+title: "Trino Summit 2022: Federating humans and data"
+author: "Brian Olsen"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-summit-2022/summit-logo.png
+---
+
+Trino has long been the de facto standard to querying large data sets over your
+cloud or on-prem storage, also known as data lakes. This Trino Summit's theme 
+instead will showcase Trino's other claim to fame: query federation. In essence,
```
Comment (mosabua) [inline _posts/2022-10-19-trino-summit-2022-teaser-2.md:11]:
```suggestion
instead will showcase Trino's other claim to fame: query federation.
```

File: _posts/2022-10-19-trino-summit-2022-teaser-2.md [hunk 15]
```diff
@@ -0,0 +1,143 @@
...
+Trino is a query engine providing an access point that exposes ANSI SQL across 
+[multiple data sources]({{site.url}}/docs/current/connector.html).
+
+I urge you to join us either in-person or virtually if you are a fan of Trino,
+big data, open source, data engineering, Java, or all the above! This conference
```
Comment (mosabua) [inline _posts/2022-10-19-trino-summit-2022-teaser-2.md:16]: all the above .. doesnt work here, right?

File: _posts/2022-10-19-trino-summit-2022-teaser-2.md [hunk 16]
```diff
@@ -0,0 +1,143 @@
...
+is free and takes place on November 10th. Check out the details below!
```
Comment (mosabua) [inline _posts/2022-10-19-trino-summit-2022-teaser-2.md:17]: do we need to add "in San Francisco" ?

Comment (mosabua) [reply _posts/2022-10-19-trino-summit-2022-teaser-2.md:17]: Also .. should we remove "Check out the details below!" .. there is already a read more link

File: _posts/2022-10-19-trino-summit-2022-teaser-2.md [hunk 17]
```diff
@@ -0,0 +1,143 @@
...
+
+<!--more-->
+
+## Register for the summit
+
+I can't help but bring up the analogy of how Trino federates heterogeneous data
+while this Trino Summit will federate many of us in the community form all
+corners of the earth. It really brings an appreciation to the international
```
Comment (mosabua) [inline _posts/2022-10-19-trino-summit-2022-teaser-2.md:25]: of the world

File: _posts/2022-10-19-trino-summit-2022-teaser-2.md [hunk 18]
```diff
@@ -0,0 +1,143 @@
...
+reach this technology has and makes me look forward to future in-person events!
```
Comment (mosabua) [inline _posts/2022-10-19-trino-summit-2022-teaser-2.md:26]:
```suggestion
reach of Trino and makes me look forward to more in-person events!
```

File: _posts/2022-10-19-trino-summit-2022-teaser-2.md [hunk 19]
```diff
@@ -0,0 +1,143 @@
...
+
+Trino Summit will be held at the Commonwealth Club in San Francisco, CA. Make
```
Comment (mosabua) [inline _posts/2022-10-19-trino-summit-2022-teaser-2.md:28]:
```suggestion
Trino Summit will be held at the Commonwealth Club in San Francisco, California. Make
```

File: _posts/2022-10-19-trino-summit-2022-teaser-2.md [hunk 20]
```diff
@@ -0,0 +1,143 @@
...
+sure you register quickly for in-person registration, as it is limited to 250
+seats! Virtual registration is also picking up quickly so register today!
```
Comment (mosabua) [inline _posts/2022-10-19-trino-summit-2022-teaser-2.md:30]:
```suggestion
seats. Virtual registration is also picking up quickly so register today!
```

File: _posts/2022-10-18-trino-summit-2022-teaser-2.md (hunk 9 above)
Comment (mosabua) [reply _posts/2022-10-18-trino-summit-2022-teaser-2.md]: ok

File: _posts/2022-10-19-trino-summit-2022-teaser-2.md [hunk 21]
```diff
@@ -0,0 +1,143 @@
...
+but none met our needs. In this session you will learn how we built a data
+architecture to collect, model, and enrich user behavior events to optimize
+Trino query performance that accelerated our ability to understand and improve
+user conversion rates.
+
+* *Roy Hasson, Head of Product at Upsolver*
+
+## Conclusion
+
+I hope you all are excited as we are to finally federate the Trino community
```
Comment (mosabua) [inline _posts/2022-10-19-trino-summit-2022-teaser-2.md:137]:
```suggestion
I hope you all are as excited as we are to finally federate the Trino community
```

**Review by mosabua — CHANGES_REQUESTED** [review]

File: _posts/2022-10-19-trino-summit-2022-teaser-2.md (hunk 15 above)
Comment (bitsondatadev) [reply _posts/2022-10-19-trino-summit-2022-teaser-2.md:16]: All the above meaning you can be a fan of Trino, big data, open source, data engineering, and Java all at once.

File: _posts/2022-10-19-trino-summit-2022-teaser-2.md (hunk 16 above)
Comment (bitsondatadev) [reply _posts/2022-10-19-trino-summit-2022-teaser-2.md:17]: Fair, I just wanted it to be clear that we'll provide more details about how to attend if someone was lazy but it should be obvious to read on.

**Review by mosabua — APPROVED** [review]
Looks good now. Let's ship this today and push it on channels tomorrow.

**Pull request merged**